### PR TITLE
added ghanaian language Twi in the person providers folder

### PR DESCRIPTION
**Pull request by Sambeth**
WIll work on other providers for same language as soon as I can.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -42,3 +43,4 @@ nosetests.xml
 .projectile
 .ropeproject
 .DS_Store
+tryout.py
```
Comment (fcurella) [inline .gitignore]: could you remove this?

File: faker/providers/person/tw_GH/__init__.py [hunk 2]
```diff
@@ -0,0 +1,592 @@
...
+        '{{first_name_female}} {{last_name_female}}',
+        '{{first_name_female}} {{last_name_female}}',
+        '{{first_name_female}} {{last_name_female}}-{{last_name_female}}',
+        '{{prefix_male}} {{first_name_male}} {{last_name_male}}',
+        '{{prefix_female}} {{first_name_female}} {{last_name_female}}',
+        '{{prefix_male}} {{first_name_male}} {{last_name_male}}',
+        '{{prefix_female}} {{first_name_female}} {{last_name_female}}'
+    )
+
+    first_names_male = (
```
Comment (fcurella) [inline faker/providers/person/tw_GH/__init__.py]: do you have any source for these names? Could you include it as comment?

**Comment by Sambeth** [comment]
The changes have been made.

**Comment by fcurella** [comment]
thanks! ✨

**Pull request merged**